TMS_Tests/Tests/TestRailTests.cs

```diff
@@ -15,8 +18,22 @@ public void SetUp()
         public void CreateTestProject()
         {
             TRDashboardPage.AddProjectButton().Click();
-            projectName = "AKaliasinskiTestProject " + DateTime.Now;
+
+            var project = new ProjectModel()
+            {
+                Name = "AKaliasinskiTestProject ",
+                Announcement = "Test announcement",
+                IsShowAnnouncement = true,
+                ProjectType = "Use a single repository for all cases (recommended)",
+                IsEnableTestCaseApprovals = true
+            };
+
+            projectName = project.Name + DateTime.Now;
```

Comment:
you can specify project name with date in the ProjectModel
example: 
 var project = new ProjectModel()
            {
                Name = "AKaliasinskiTestProject " + DateTime.Now,
                ...

Reply:
Updated